Test/TemplateSourceTests.cs

```diff
@@ -42,45 +43,38 @@ public void TestNginxMimeTypes()
         }
 
         [TestMethod]
-        public void TestMimeDbMimeTypes()
+        public async Task TestMimeDbMimeTypesAsync()
         {
-            var keyPairs = GetMimeTypesFromJson(MIMEDB_URL, resource =>
+            var keyPairs = await GetMimeTypesFromJsonAsync(MIMEDB_URL, resource =>
             {
-                return from mimeTypes in resource.Children()
-                       let mimeType = ((JProperty) mimeTypes).Name
-                       from mimeTypeProperties in mimeTypes.Children()
-                       from mimeTypeProperty in mimeTypeProperties.Children()
-                       from mimeTypeValue in mimeTypeProperty.Values()
-                       where (mimeTypeProperty as JProperty)?.Name == "extensions"
-                       select new[]
-                       {
-                           mimeType,
-                           mimeTypeValue.Value<string>()
-                       };
+                return resource.RootElement.EnumerateObject()
```

Comment:
why this syntax change?

Reply:
Linq method syntax seems cleaner and shorter than query syntax.

Reply:
Reverted. Opened separate issue.